### PR TITLE
Add Project Reference report and refactor helper message

### PR DESCRIPTION
**Pull request by Chrisliang96**

### CODE REVIEW

File: src/ChangeDB.ConsoleApp/Commands/DumpSql.cs [hunk 1]
```diff
@@ -6,54 +6,50 @@
...
-        [Value(4, MetaName = "output-file", Required = true, HelpText = "output script file")]
+        [Value(4, MetaName = "Output-FilePath", Required = true, HelpText = "Enter the file path to saved transformed sql scripts")]
         public string TargetScript { get; set; }
 
-        [Option('f', "force", HelpText = "drop target sql script file if exists", Default = false)]
+        [Option('f', "force", HelpText = "Warning:This option will DROP target sql script file if exists in path", Default = false)]
         public bool DropTargetDatabaseIfExists { get; set; } = false;
 
-        [Option("dump-scope", HelpText = "(All/Metadata/Data)", Default = MigrationScope.All)]
+        [Option("Dump-Option", HelpText = "Identify which part of database you want to transform. (ej:All/Metadata/Data)", Default = MigrationScope.All)]
```
Comment (Chrisliang96) [inline src/ChangeDB.ConsoleApp/Commands/DumpSql.cs]: I wonder whether here i can set option name to up-grader name, like "dump-xxx" to "Dump-Xxx",will it mke user inconvenient

File: src/ChangeDB.ConsoleApp/Commands/Migration.cs [hunk 2]
```diff
@@ -9,47 +9,42 @@ namespace ChangeDB.ConsoleApp.Commands
...
-        [Option("migration-scope", HelpText = "(All/Metadata/Data)", Default = MigrationScope.All)]
+        [Option("Migration-Option", HelpText = "Identify which part of database you want to migration. (ej:All/Metadata/Data)(All/Metadata/Data)", Default = MigrationScope.All)]
         public MigrationScope MigrationScope { get; set; } = MigrationScope.All;
 
-        [Option("name-style", HelpText = "target object name style (Original/Lower/Upper).")]
+        [Option("Naming-Style", HelpText = "Identify the naming style of transformed database scripts. (ej:Original/Lower/Upper).")]
         public NameStyle NameStyle { get; set; } = NameStyle.Original;
 
-        [Option("max-fetch-bytes", HelpText = "max fetch bytes when read source database (unit KB), default value is 100 (100KB).")]
+        [Option("Max-Fetch-Capacity", HelpText = "Enter the max capacity when ChangeDB fetch data from source database, default value is 100 (100 here equals 100KB).")]
```
Comment (obpt123) [inline src/ChangeDB.ConsoleApp/Commands/Migration.cs]: Change the option name will 
impact on existing users.

File: src/ChangeDB.ConsoleApp/Commands/Migration.cs [hunk 3]
```diff
@@ -9,47 +9,42 @@ namespace ChangeDB.ConsoleApp.Commands
...
 
-        [Option("target-default-schema",
-            HelpText = "target database default schema.")]
+        [Option("target-default-schema", HelpText = "target database default schema.")]
         public string TargetDefaultSchema { get; set; }
 
 
-        [Option("hide-progress",
-           HelpText = "hide progress bar.", Default = false)]
+        [Option("Hide-Process-Indicator", HelpText = "Set ture console table will hide transfrom indicator while ChangeDB migrate database. default value is false.", Default = false)]
```
Comment (obpt123) [inline src/ChangeDB.ConsoleApp/Commands/Migration.cs]: Change the option name will impact on existing users.

**Review by obpt123 — COMMENTED** [review]
Change the option name is dangerous for existing user.

**Review by Chrisliang96 — COMMENTED** [review]
already recover option names

**Pull request merged**